modules/gapi/src/compiler/passes/pattern_matching.cpp

```diff
@@ -0,0 +1,385 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2019 Intel Corporation
+
+#include <unordered_set>
+
+#include "pattern_matching.hpp"
+
+cv::gapi::SubgraphMatch cv::gapi::findMatches(cv::gimpl::GModel::Graph patternGraph, cv::gimpl::GModel::Graph compGraph) {
```

Comment:
OpenCV coding guideline says:

> Code lines should not be very long. Normally, they should be limited to 100 characters.

This particular line is 122 characters long, and I believe it is not the only one.

In G-API, our regular coding style is usually about 70-80 characters per line (see other files except kernels).
Please reformat your code to conform to that guidelines.

Reply:
Code is aligned to 100 characters per line (for 1st time) in pattern_matching.cpp file besides one comment, responsible for picture.
Have seen other files, they look like also be aligned for that limit (or just fit in this limit, but not 80)